packages/p2p/src/components/section-error/section-error.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+
+type TSectionErrorProps = {
```

Comment:
```suggestion
type TSectionError = {
```

Reply:
isnt the naming convention T then Props? cause in other components it should be that right?

Reply:
In most of our ts files we don't use this format there are some developers who uses the prop along the type name, but AFAIK we should not have it